### ISSUE TITLE
Rohrer Bug: Double outlines are confusing

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#7971]
"Now I need to chop more.  There are yellow outlines around chopping targets but very similar-looking white outlines around drop-location targets for the logs that build up in my hand as I chop.  As it gets dark, I'm looking on the ground for more wood to chop, and the outlines are confusing."

Not sure the solution for this one.  May need to brainstorm it.

**Comment by metachronism** [comment]
I dont think this is really accurate.  block placing previews dont show up if you are targeting anything
